blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How a hack week project made Sourcegraph.com easier to use, by searching the top 10k open source repositories on GitHub by default."
+author: Issac Trotts, Software Engineer at Sourcegraph
+publishDate: 2019-08-16T00:00-11:55
+tags: [
+  blog
+]
+slug: Search-the-top-10000-open-source-repositories-Sourcegraph-com
+heroImage: /blog/sourcegraph-com-usage.png
+published: true
+---
+
+<p style="text-align: center">
+  <img src="/blog/sourcegraph-com-usage.png" alt="My top feature request is: no new features. Just fix what's broken">
+  <br/><small>Sourcegraph.com stats as of August 2019</small>
+</p>
+
+Wouldn't it be awesome if you could review, browse, and search code from the top ten thousand open source repositories on GitHub? You can, using [Sourcegraph.com](https://sourcegraph.com), and we've now made it easier to use!
```

Comment:
This makes me think that I can only review and browse code from the top 10k repos (which isn't true).

It is also currently phrased in a way that all this was already possible before and it is now easier (which doesn't seem accurate).

Reply:
Changed wording in https://github.com/sourcegraph/about/pull/217/commits/11115708c045a6edd7062929380851f0b7e57388 to "Wouldn't it be awesome if you could scope a code search to find usage examples from the top ten thousand open source repositories on GitHub*"